packages/design-system/src/styles/components/_Table.scss

```diff
@@ -5,6 +5,85 @@ $table-striped-background-color: $color-gray-lightest !default;
 $table-striped-header-background-color: $color-gray-lightest !default;
 // Max-width of the header column for a stacked table
 $table-stacked-header-max-width: 160px !default;
+$table-padding: $spacer-2 !default;
+$table-dense-padding: $spacer-half $spacer-2 !default;
+
+.ds-c-table {
+  border-collapse: collapse;
+  border-spacing: 0;
+  margin: 0;
+
+  th,
+  thead td {
+    background-color: $table-header-background-color;
+    text-align: left;
+  }
+
+  tbody th {
+    font-weight: $font-normal;
+  }
+
+  th,
+  td {
+    border: 1px solid $color-gray;
+    padding: $table-padding;
+  }
+}
+
+.ds-c-table--borderless {
+  thead {
+    th,
+    tr {
+      background-color: transparent;
+    }
+
+    th {
+      border-top: 0;
+    }
+  }
+
+  th,
+  td {
+    border-left: 0;
+    border-right: 0;
+  }
+}
+
+.ds-c-table--dense {
+  th,
+  td {
+    padding: $table-dense-padding;
+  }
+}
+
+.ds-c-table--striped {
+  thead {
+    th,
+    tr {
+      background-color: $table-striped-header-background-color;
+    }
+  }
+
+  tbody {
+    tr:nth-child(even) {
+      background-color: $table-striped-background-color;
+    }
+
+    tr:nth-child(odd)::before {
+      background-color: transparent;
+    }
+  }
+}
+
+.ds-c-table__wrapper {
+  overflow-x: auto;
+}
+
+.ds-c-table__caption {
+  font-weight: $font-bold;
+  margin-bottom: $spacer-1;
```

Comment:
If there's already a `margin-bottom` here, can we just remove the margin utility from the default scrollable notice?

Reply:
The spacing is less, but workable. What do you think?

Current:
![Screen Shot 2020-11-04 at 4 04 44 PM](https://user-images.githubusercontent.com/20322880/98168226-44156280-1eb8-11eb-92ab-3c91cea58b99.png)

Removed margin-bottom from scrollable notice: ![Screen Shot 2020-11-04 at 4 05 10 PM](https://user-images.githubusercontent.com/20322880/98168186-2cd67500-1eb8-11eb-87df-26a376aedcbb.png)